Abhi-AnimeX/app/src/main/java/com/abhijith/animex/ui/navigation/Screen.kt

```diff
@@ -0,0 +1,6 @@
+package com.abhijith.animex.ui.navigation
+
+sealed class Screen(val route: String) {
+    data object AnimeList : Screen("animeList")
+    data object AnimeDetails : Screen("animeDetails/{selectedAnimeItem}")
```

Comment:
why not enum?

Reply:
An enum would be sufficient if all screens had the same structure (just a route string), but the sealed class allow us to define different parameters for each subtype. @anandwana001